control-station/src/components/ControlPanel/style.css

```diff
@@ -0,0 +1,30 @@
+#controlpanel{
+    position: fixed;
+    bottom: 0;
+    width: 100%;
+    background-color: black;
+    height: 9%;
```

Comment:
Suggestion [repeating, also for Navbar]: using a percentage for height makes the element too short on narrower views. Consider relying on the content and padding to determine the height instead of explicitly setting it.